matrimony_admin/views.py

```
@@ -162,21 +196,21 @@ class FinancialManagement(TemplateView):
 class NotifcationManagement(FormView):
     template_name = "notification_management.html"
     form_class = NotificationDetailsForm
-    success_url = 'notification_management'
-
+    success_url = "notification_management"
 
     def post(self, request: HttpRequest, *args: str, **kwargs: dict) -> HttpResponse:
         form = self.form_class(request.POST)
         if form.is_valid():
             notification = form.save(commit=True)
```

Ruff

Local variable `notification` is assigned to but never used

Remove assignment to unused variable `notification`

(F841)